net/ddns-scripts/files/usr/lib/ddns/update_dnspod_cn.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/sh
+
+# Check inputs
+[ -z "$username" ] && write_log 14 "Configuration error! [User name] cannot be empty"
+[ -z "$password" ] && write_log 14 "Configuration error! [Password] cannot be empty"
+
+# Check external tools
+[ -n "$CURL_SSL" ] || write_log 13 "Dnspod communication require cURL with SSL support. Please install"
+[ -n "$CURL_PROXY" ] || write_log 13 "cURL: libcurl compiled without Proxy support"
+
+# Declare variables
+local __URLBASE __HOST __DOMAIN __TYPE __CMDBASE __POST __POST1 __RECIP __RECID __TTL
```

Comment:
As far as I know ash does not allow a `local` definition without a function

Reply:
As not really needed, I just comment out the line.
(See similar usage in other scripts in the same folder, maybe need a bigger rework)